src/main/kotlin/eu/europa/ec/eudi/sdjwt/SdJwtVerifier.kt

```diff
@@ -119,7 +119,7 @@ fun interface JwtSignatureVerifier {
      * @return a new [JwtSignatureVerifier] that in addition applies to the
      *  extracted payload the [additionalCondition]
      */
-    fun and(additionalCondition: (Claims) -> Boolean): JwtSignatureVerifier = JwtSignatureVerifier { jwt ->
+    fun and(additionalCondition: suspend (Claims) -> Boolean): JwtSignatureVerifier = JwtSignatureVerifier { jwt ->
```

Comment:
Do we really need a suspended lamda here?

Reply:
Not necessarily, but I added it for API flexibility.